pkg/shell/superuser.tsx

```diff
@@ -149,15 +149,15 @@ const UnlockDialog = ({
             if (proxy.Methods) {
                 const ids = Object.keys(proxy.Methods);
                 if (ids.length == 0)
-                    start("sudo");
+                    start("run0");
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22373-facd81b1-20250827-151151-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)